app/src/main/java/com/example/hadar/trempyteam/searchTrempFragment.java

```diff
@@ -132,10 +128,39 @@ public void onClick(View v) {
 
                 }
             });
-        }
+
         return rootView;
     }
 
+    public LatLng getLocationFromAddress(Context context, String strAddress)
+    {
+        Geocoder coder= new Geocoder(context);
+        List<Address> address;
+        LatLng p1 = null;
+
+        try
+        {
+            address = coder.getFromLocationName(strAddress, 5);
+            if(address==null)
+            {
+                return null;
+            }
+            Address location = address.get(0);
+            location.getLatitude();
```

Comment:
What are these statements for? Get the coordination then assign it to nothing as well as do nothing.